wgengine/magicsock/magicsock.go

```diff
@@ -2599,86 +2595,91 @@ func (c *Conn) initialBind() error {
 	return nil
 }
 
-func (c *Conn) listenPacket(ctx context.Context, network, host string, port uint16) (net.PacketConn, error) {
+// listenPacket opens a listener. Host must be "" or an IP address.
+func (c *Conn) listenPacket(network, host string, port uint16) (net.PacketConn, error) {
+	ctx := context.Background() // unused without DNS name to resolve
 	addr := net.JoinHostPort(host, fmt.Sprint(port))
 	if c.packetListener != nil {
 		return c.packetListener.ListenPacket(ctx, network, addr)
 	}
 	return netns.Listener().ListenPacket(ctx, network, addr)
 }
 
-func (c *Conn) bind1(ruc **RebindingUDPConn, which string) error {
+// bind1 binds a UDP socket.
+// If there was an existing UDP socket bound, it closes it.
+// The caller is responsible for informing the portMapper of any changes.
+func (c *Conn) bind1(rucPtr **RebindingUDPConn, which string) error {
```

Comment:
`which` isn't documented. say that it's "udp4" or "udp6"? (those are the only two valid values, right?)

Reply:
Renamed `which` to `network` to match package net's terminology and documented.